common/ch_channel_map/oracular

```diff
@@ -0,0 +1,18 @@
+# NOTE(ganso): This was copy-pasted from noble
+
+# These are charmhub mappings for charms that are outside of the main large
+# groupings like openstack and ceph etc.
+
+# Versions are based on https://docs.openstack.org/charm-guide/latest/project/charm-delivery.html
+
+# "Charm Delivery" specifies vault 1.8 for jammy, however field have actually been using 1.7.
+# Stick with 1.7 to more closely replicate customer deployments.
+CHARM_CHANNEL[vault]=latest/edge
+
+CHARM_CHANNEL[rabbitmq-server]=latest/edge
+CHARM_CHANNEL[mysql-router]=latest/edge
+CHARM_CHANNEL[mysql-innodb-cluster]=latest/edge
+CHARM_CHANNEL[hacluster]=2.4/edge
+CHARM_CHANNEL[ovn-central]=23.09/edge # FIXME: update once caracal released
```

Comment:
need to fix these since the correct version is 24.03. I will submit a separate patch.

Reply:
https://github.com/canonical/stsstack-bundles/pull/299